src/server.ts

```diff
@@ -27,46 +39,124 @@ const blobServiceClient = BlobServiceClient.fromConnectionString(
   config.STORAGE_CONNECTION_STRING
 );
 
-if (!config.isProduction) {
-  initializeCompaniesBlob(
-    blobServiceClient,
-    config.CONTAINER_NAME,
-    config.BLOB_NAME
-  )()
-    // tslint:disable-next-line: no-console
-    .then(() => console.log("Test Blob initialized"))
-    // tslint:disable-next-line: no-console
-    .catch(console.log);
-}
+const attributeAuthorityPostgresDb = new Sequelize(
+  config.ATTRIBUTE_AUTHORITY_POSTGRES_DB_URI,
+  sequelizePostgresOptions()
+);
 
-server.post<{ Body: GetCompaniesBody; Response: Companies }>(
-  "/companies",
+// Initialize models and sync them
+initModels(attributeAuthorityPostgresDb);
+
+server.get<{
+  Querystring: IGetOrganizationsQueryString;
+  Response: Organizations;
+}>(
+  "/organizations",
   {
     preHandler: async (request, reply) =>
       withRequestMiddlewares(
         request,
         reply,
-        requiredBodyMiddleware(GetCompaniesBody)
-      )
+        queryParamsMiddleware(IGetOrganizationsQueryString)
+      ),
+  },
+  organizationHandler.getOrganizationsHandler()
+);
+
+server.post<{ Body: OrganizationWithReferents }>(
+  "/organizations",
+  {
+    preHandler: async (request, reply) =>
+      withRequestMiddlewares(
+        request,
+        reply,
+        requiredBodyMiddleware(OrganizationWithReferents)
+      ),
+  },
+  organizationHandler.upsertOrganizationHandler()
+);
+
+server.get(
+  "/organization/:keyOrganizationFiscalCode",
+  {
+    preHandler: async (request, reply) =>
+      withRequestMiddlewares(
+        request,
+        reply,
+        pathParamsMiddleware(KeyOrganizationFiscalCode)
+      ),
+  },
+  organizationHandler.getOrganizationHandler()
+);
+
+server.delete(
+  "/organization/:keyOrganizationFiscalCode",
+  {
+    preHandler: async (request, reply) =>
+      withRequestMiddlewares(
+        request,
+        reply,
+        pathParamsMiddleware(KeyOrganizationFiscalCode)
+      ),
   },
-  getCompaniesHandler(
-    blobServiceClient,
-    config.CONTAINER_NAME,
-    config.BLOB_NAME
-  )
+  organizationHandler.deleteOrganizationHandler()
 );
 
-server.post<{ Body: UserCompanies }>(
-  "/user",
+server.get(
+  "/organization/:keyOrganizationFiscalCode/referents",
   {
     preHandler: async (request, reply) =>
       withRequestMiddlewares(
         request,
         reply,
-        requiredBodyMiddleware(UserCompanies)
-      )
+        pathParamsMiddleware(KeyOrganizationFiscalCode)
+      ),
+  },
+  referentHandler.getReferentsHandler()
+);
+
+server.post<{ Body: ReferentFiscalCode }>(
+  "/organization/:keyOrganizationFiscalCode/referents",
+  {
+    preHandler: async (request, reply) =>
+      withDoubleRequestMiddlewares(
+        request,
+        reply,
+        pathParamsMiddleware(KeyOrganizationFiscalCode),
+        requiredBodyMiddleware(ReferentFiscalCode)
+      ),
+  },
+  referentHandler.insertReferentHandler()
+);
+
+server.delete<{ Body: ReferentFiscalCode }>(
```

Comment:
ditto from api specs

Reply:
in https://github.com/pagopa/ade-aa-ms-mock/pull/16/commits/9bd4e8454fe620851cd808de302de858df4fb62c